app/src/main/java/com/kamron/pogoiv/logic/PokemonNameCorrector.java

```diff
@@ -19,19 +18,19 @@
 public class PokemonNameCorrector {
     private final PokeInfoCalculator pokeInfoCalculator;
     private final HashMap<String, String> userCorrections;
-    /* We don't want memory usage to get out of hand for stuff that can be computed. */
-    private final LruCache<String, PokeDist> cachedCorrections;
 
     public PokemonNameCorrector(PokeInfoCalculator pokeInfoCalculator, Map<String, String> storedUserCorrections) {
         this.pokeInfoCalculator = pokeInfoCalculator;
         userCorrections = new HashMap<>(pokeInfoCalculator.getPokedex().size());
         userCorrections.putAll(storedUserCorrections);
-        userCorrections.put("Sparky", pokeInfoCalculator.get(132).name);
-        userCorrections.put("Rainer", pokeInfoCalculator.get(132).name);
-        userCorrections.put("Pyro", pokeInfoCalculator.get(132).name);
```

Comment:
Did you mean to drop changes to userCorrections? If not, as I guess, would you restore them?